### PR TITLE
fix: log records bug fixes

### PR DESCRIPTION
**Pull request by itssharmasandeep**
## Description
- Bug fixes in marker indicator.
    1. Wrong number of logs are there in the tooltip for marker (on hover)
    2. Wrong summary is there in the tooltip for marker (on hover)
- New column span_id in log records table

### Checklist:
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/distributed-tracing/src/shared/dashboard/widgets/waterfall/waterfall/waterfall-chart.component.ts [hunk 1]
```diff
@@ -202,7 +202,7 @@ export class WaterfallChartComponent implements OnChanges {
 
     let markerData: MarkerTooltipData = {
       relativeTimes: [],
-      summary: spanWaterfallData.logEvents[0].summary
+      summary: spanWaterfallData.logEvents.find(logEvent => logEvent.timestamp === marker.timestamps[0])?.summary ?? ''
```
Comment (anandtiwary) [inline projects/distributed-tracing/src/shared/dashboard/widgets/waterfall/waterfall/waterfall-chart.component.ts:205]: Could you please explain the bugs in the description or add any github issue if present?

File: projects/components/src/sequence/renderer/sequence-bar-renderer.service.ts [hunk 2]
```diff
@@ -230,7 +230,7 @@ export class SequenceBarRendererService {
               ? scaledEnd - this.markerWidth - 2
               : marker.markerTime
         });
-        markerTime = scaledNormalizedMarkers[index].markerTime;
```
Comment (anandtiwary) [inline projects/components/src/sequence/renderer/sequence-bar-renderer.service.ts:233]: why do we not need normalized markers?

**Comment by anandtiwary** [comment]
Looks good apart from the two questions

File: projects/distributed-tracing/src/shared/dashboard/widgets/waterfall/waterfall/waterfall-chart.component.ts (hunk 1 above)
Comment (itssharmasandeep) [reply projects/distributed-tracing/src/shared/dashboard/widgets/waterfall/waterfall/waterfall-chart.component.ts:205]: Done!!

File: projects/components/src/sequence/renderer/sequence-bar-renderer.service.ts (hunk 2 above)
Comment (itssharmasandeep) [reply projects/components/src/sequence/renderer/sequence-bar-renderer.service.ts:233]: This is just the terminology I have used, 
No other significance of it than grouping the scaled markers
If you could suggest the better naming for this?

**Review by anandtiwary — APPROVED** [review]

**Pull request merged**